app/api_topologies.go

```diff
@@ -80,21 +80,24 @@ func updateFilters(rpt report.Report, topologies []APITopologyDesc) []APITopolog
 		ns = append(ns, namespace)
 	}
 	sort.Strings(ns)
-	for i, t := range topologies {
-		if t.id == podsID || t.id == servicesID || t.id == deploymentsID || t.id == replicaSetsID {
-			topologies[i] = updateTopologyFilters(t, []APITopologyOptionGroup{
-				kubernetesFilters(ns...), unmanagedFilter,
-			})
+	if len(ns) > 0 {
+		// We only want to apply k8s filters if any k8s info is present
```

Comment:
I would rephrase this into _We only want to apply k8s filters for k8s-relevant topologies_ (since you may be showing the filters even without k8s-tracked nodes)